aiidalab_ispg/spectrum.py

```diff
@@ -507,6 +521,23 @@ def _plot_spectrum(
             if self.conformer_toggle.value:
                 self._plot_conformer(x, y, conf_id, update=False)
 
+        # Energy unit not nm needs converting for spectrum analysis
+        if energy_unit != EnergyUnit.NM:
+            total_cross_section_nm = np.zeros(Spectrum.N_SAMPLE_POINTS)
+
+            for conf_id, conformer in enumerate(self.conformer_transitions):
+                spec = Spectrum(conformer["transitions"], conformer["nsample"])
```

Comment:
I don't think we need to create a new instance here, we can reuse the one from above.
```suggestion
```